include/z64player.h

```diff
@@ -1213,8 +1222,8 @@ typedef struct Player {
     /* 0xAA0 */ f32 closestSecretDistSq; // Used to augment `secretRumbleCharge`. Cleared every frame
     /* 0xAA4 */ s8 unk_AA4;
     /* 0xAA5 */ u8 unk_AA5; // PlayerUnkAA5 enum
-    /* 0xAA6 */ u16 unk_AA6; // flags of some kind
```

Comment:
Why are we keeping the `unk_AA6` part of the name now that we know what the flags are doing? is it just because there is unused or unknown bit space as the data is u16? The fact that the whole flag variable gets set to zero if all rotation is centered kinda suggests the other bits aren't used for something else right?

Reply:
This is a temporary measure until we think of a good name. In OoT, the PR originally called these "noRestFlags" but the name wasnt particularly popular, so it was left as unk.

If you have a name suggestion feel free to drop it. I will probably try to come up with my own name when I get to this area of the code in oot player docs.

Reply:
Yeah, we do know what all the flags are doing, it's just that we haven't been able to reach a consensus on a name for it yet.